.gitignore

```diff
@@ -4,4 +4,4 @@ npm-debug.log
 /.env
 .vscode/
 yarn-error.log
-
+.idea/
```

Comment:
why do we need this?

Reply:
If anyone is using WebStorm IDE, they might find it useful to block `.idea` folder which is generated by the IDE. If you think it is pointless to add to the whole project, then I am happy to remove it.

Reply:
Oh I had no idea (no pun intended), never used WebStorm. Let's keep it then!